### ISSUE TITLE
Update Editor State Without De-initialising

### ISSUE THREAD

**Issue by thombruce** [thombruce/articles#4]
This is definitely possible, I had a tab open about the process... but I have either closed it or it was on my mobile and I'll... maybe I'll find it in history.

It's a relatively new method that was added to editor whereby the content can be replaced on the fly.

Doing this should eliminate the stutter currently present when switching between documents.

**Comment by thombruce** [comment]
Simply removing the nullification of article before nav (22965ca15088ebc9ca365c097feaff329df02d5a) removes the stutter. This was originally added to ensure that the editor state changed, but a `:key` was added to the form to ensure that instead (which works, provided the article is updated beforeRouteUpdate - and not just in the mounted step).

What it introduced, however, are errors received from Tiptap editor about being unable to append a node...

...which... is perhaps indicative of the fact that `article` and therefore the document is being updated before the call to deinit and re-initialise the editor.

I experimented with an update Document call a little on Tiptap but this... triggered my save functionality and overwrote the current article. So... that was based on a watch hook on the document itself, so either... that needs fixing or, perhaps by checking against ID as well as doc state... or... something else, like, maybe the way this is set up needs a more dramatic rewrite.
